public/app/views/ICE/src/method-draw.js

```diff
@@ -3495,6 +3515,10 @@ var SOTP = 0;
           {sel:'#tool_save', fn: function() { editingsource ? saveSourceEditor(): clickSave() }, evt: 'mouseup', key: [modKey + 'S', true]},
           {sel:'#tool_export', fn: clickExport, evt: 'mouseup'},
           {sel:'#tool_open', fn: clickOpen, evt: 'mouseup'},
+          {sel:'#tool_help', fn: clickHelp, evt: 'mouseup'},
+          {sel:'#tool_about', fn: clickAbout, evt: 'mouseup'},
+          {sel:'.tool_import', fn: clickImport, evt: 'mousedown'},
```

Comment:
Could you have meant to refer .tool_import and .tool_import_mobile by id instead (#)?